API/PROBLEMS/with_identifiers/strange

```diff
@@ -0,0 +1,12 @@
+--------------------------------------------------------------------------------
+
+INSTANCES
+
+- Namegen.next_ident_away
+- Term.types (* plural: misleading *)
+- EConstr.types (* plural: misleading *)
+- Term.Unsafe.equal_nouniv
```

Comment:
I can't find this `Term.Unsafe` module or the `equal_nouniv` function, where are they? https://github.com/coq/coq/search?q=equal_nouniv&type=Code&utf8=%E2%9C%93